proto.list

```diff
@@ -0,0 +1,12 @@
+google/api/http.proto
```

Comment:
This file is actually not needed (it's used in the process of generation but is not used by the library itself). Let's `git rm` it and I'll update the generator to erase it.

Reply:
https://github.com/googleapis/gapic-generator-typescript/pull/91